tests/baselines/reference/reverseMappedTypeContextualTypeNotCircular.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/compiler/reverseMappedTypeContextualTypeNotCircular.ts(10,3): error TS2322: Type '(state: any, props: any) => {}' is not assignable to type 'Selector<unknown, {}>'.
+tests/cases/compiler/reverseMappedTypeContextualTypeNotCircular.ts(10,3): error TS2322: Type '(state: any, props: any) => {}' is not assignable to type 'Selector<any, {}>'.

```

Comment:
this is an expected change as now `/*1*/` became an inference source that can create candidates for `S`:
```ts
const mapStateToProps = createStructuredSelector({
  editable: (state: /*1*/any, props: any) => editable()
});
```